### PR TITLE
Add static content pages

### LINKED ISSUE #50: Add some more static content
- [x] Main index page
    - [x] Add new architecture diagram
    - [ ] Add example comparisons when available
- [x] Contact us page
    - [x] Add Michael to this
    - [x]  Get the "you don't have permission to create models" message to redirect here
- [x] Privacy policy (see https://github.com/ModellingWebLab/deployment/issues/4)
- [x] Background info page similar to old about page, with pointer to paper on new Web Lab, etc
- [x] 'For authors' hints on unexpected results

Child of #68.

### PR DESCRIPTION
**Pull request by jonc125**
This branch is currently deployed on [scrambler](https://scrambler.cs.ox.ac.uk/) so you can easily review the text @mirams @MichaelClerx. Let me know if you'd like any changes!

Fixes #50.

### CODE REVIEW

File: weblab/templates/home.html [hunk 1]
```diff
@@ -1,5 +1,72 @@
...
+
+  <p>
+    This is the beta version of 'Web Lab 2', a second generation Web Lab incorporating experimental data and parameter fitting.
+    You can read more about our plans in a <a href="https://www.biorxiv.org/content/early/2018/01/31/257683">preprint of our next Web Lab paper</a>.
+  </p>
+
+  <h2>Quick start links</h2>
+  <ul>
+    <li><a href="{% url 'experiments:list' %}">View results of experiments</a> stored on this site.</li>
+    <li>Compare the results of different experiments, e.g. ...; or <a href="{% url 'experiments:list' %}">set up your own comparisons</a>.</li>
```
Comment (MichaelClerx) [inline weblab/templates/home.html]: What's missing here?

File: weblab/templates/home.html [hunk 2]
```diff
@@ -1,5 +1,72 @@
...
+  <p>
+    We have been trying to do the same thing with 'protocols' - to define what you have to
+    do to replicate/simulate an experiment, and analyse the results.
+    We can then curate models according to their functional behaviour under a range of experimental scenarios.
+  </p>
+  <p>
+    For the first time, we can thus easily compare how different models react to the same protocol,
+    or compare how a model behaves under different protocols.
+  </p>
+  <p class="right"><a href="{% url 'about' %}">Read more...</a></p>
```
Comment (MichaelClerx) [inline weblab/templates/home.html]: Maybe not the place, but could our links be a more prominent shade of blue? The current one is too close to the black text on my screen

File: weblab/templates/home.html [hunk 3]
```diff
@@ -1,5 +1,72 @@
...
+    {% else %}
+      {% url 'accounts:register' as analyse_link %}
+    {% endif %}
+    <li><a href="{{ analyse_link }}">Analyse your own models/protocols</a>.</li>
+    <li>Find out more about this site - read on!</li>
+  </ul>
+
+  <h2>What is the Web Lab?</h2>
+  <p>
+    Modellers have adopted XML-based markup languages to describe mathematical models over the past decade.
```
Comment (MichaelClerx) [inline weblab/templates/home.html]: languages (e.g. CellML)

The only SBML electrophysiology model I've seen is a Hodgkin-Huxley implementation

File: weblab/templates/home.html [hunk 4]
```diff
@@ -1,5 +1,72 @@
...
+  <p>
+    For the first time, we can thus easily compare how different models react to the same protocol,
+    or compare how a model behaves under different protocols.
+  </p>
+  <p class="right"><a href="{% url 'about' %}">Read more...</a></p>
+
+  <p>
+    This website is our prototype Web Lab for the domain of cardiac electrophysiology.
+    It brings together models encoded using <a href="http://www.cellml.org/">CellML</a> and
+    virtual experiment protocols encoded in <a href="https://chaste.cs.ox.ac.uk/trac/wiki/FunctionalCuration/ProtocolSyntax">our own language</a>,
```
Comment (MichaelClerx) [inline weblab/templates/home.html]: I think eventually the FC syntax (or whatever we choose to use) should be explained on this website, but OK for now

File: weblab/templates/about.html [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  <p>
+    The <a href="https://www.sciencedirect.com/science/article/pii/S0006349515047530">initial paper about the Web Lab</a> was published in <i>Biophysical Journal</i>.
+    <!-- A <a href="https://peerj.com/preprints/1338/">preprint version is available from PeerJ</a>. -->
+    A <a href="https://www.biorxiv.org/content/early/2018/01/31/257683">preprint of our next Web Lab paper</a> sets out our plans for the next two years.
+  </p>
+  <p>
+    For more background on the 'Functional Curation' experiment execution software, our <a href="http://dx.doi.org/10.1016/j.pbiomolbio.2011.06.003">2011 reference publication</a> is a good place to start.
+    We also have a <a href="http://dx.doi.org/10.1016/j.pbiomolbio.2014.10.001">paper on the underlying concept of <i>virtual experiments</i></a>
+    (along with an <a href="https://peerj.com/preprints/273/">open access preprint</a>).
+    Full details about the <a href="https://chaste.cs.ox.ac.uk/trac/wiki/FunctionalCuration">open source software implementing the back-end for this system</a> can be found on the Chaste website.
```
Comment (MichaelClerx) [inline weblab/templates/about.html]: Is this still relevant with the new Python back-end?

File: weblab/templates/home.html (hunk 3 above)
Comment (jonc125) [reply weblab/templates/home.html]: We talk specifically about CellML further down. I'd rather keep this general, as the Web Lab can be used for non-cardiac domains too.

File: weblab/templates/home.html (hunk 4 above)
Comment (jonc125) [reply weblab/templates/home.html]: Indeed.

File: weblab/templates/about.html (hunk 5 above)
Comment (jonc125) [reply weblab/templates/about.html]: The Python backend is also documented there. But at the moment scrambler is running the original C++ backend.

File: weblab/templates/home.html (hunk 2 above)
Comment (jonc125) [reply weblab/templates/home.html]: Done

**Comment by jonc125** [comment]
@mirams any change requests?

**Comment by mirams** [comment]
Maybe just some idiot proof documentation about how to update and deploy changes to the template text somewhere?

**Comment by mirams** [comment]
Looks good!

**Pull request merged**